src/nix/store-info.cc

```diff
@@ -29,21 +29,21 @@ struct CmdPingStore : StoreCommand, MixJSON
             if (auto version = store->getVersion())
                 notice("Version: %s", *version);
             if (auto trusted = store->isTrustedClient())
-                notice("Trusted: %s", *trusted);
+                notice("Trusted: %s", *trusted ? "true" : "false");
         } else {
             nlohmann::json res;
             Finally printRes([&]() {
-                logger->cout("%s", res);
+                logger->cout("%s", res.dump(2));
```

Comment:
What is this change? formatting the JSON?

Reply:
This change is about formatting the json like in the screenshot below:
![image](https://user-images.githubusercontent.com/24916780/267918130-baaf7bc2-40c5-4ee1-a7fa-f6f178a6f6db.png)

Without the change, the json looked like: 
```
{"trusted": true, "url": "local", "version": "2.1.0"}
```
I added the code to view the json with some indents ans spaces to give a better to view for this json command.

Reply:
We do not usually format output json, so I for consistency I don't think we want to here either. If we decide to do so, it should be across the board not a one-off decision for one command.

Reply:
ok.  I will revert the changes

Reply:
@Ericson2314  the changes related to aliasing which was done in `args.cc` and store info json formatting were reverted.  i will try the new approach you mentioned for aliasing in `main.cc`.